### PR TITLE
fix:Corrected From mail while sending violation in Violation details

### PR DESCRIPTION
**Pull request by sidharthjain-zemoso**
# Description
1. Corrected From mail while sending violation in Violation details

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List
any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

**Review by santhosh-challa — DISMISSED** [review]

File: webapp/src/config/configurations.ts [hunk 1]
```diff
@@ -54,13 +54,13 @@
          logout: '',
 
          CloudformationTemplateUrl: ''
-       }
+      }
```
Comment (mkinchin-paladin) [inline webapp/src/config/configurations.ts:57]: revert

File: webapp/src/app/pacman-features/secondary-components/pacman-policy-violations/pacman-policy-violations.component.ts [hunk 2]
```diff
@@ -123,8 +123,16 @@ export class PacmanPolicyViolationsComponent implements OnInit, OnDestroy {
...
+      const severityOrder = ["critical", "high", "medium", "low"];
+      
       const AStatus = a["Status"].valueText.toLowerCase();
       const BStatus = b["Status"].valueText.toLowerCase();
+      const ASeverity = a["Severity"].valueText.toLowerCase();
+      const BSeverity = b["Severity"].valueText.toLowerCase();
+
+      if(order.indexOf(AStatus)==order.indexOf(BStatus)){
+        return (severityOrder.indexOf(ASeverity) < severityOrder.indexOf(BSeverity) ? -1 : 1) * (isAsc ? 1 : -1);
+      }
```
Comment (mkinchin-paladin) [inline webapp/src/app/pacman-features/secondary-components/pacman-policy-violations/pacman-policy-violations.component.ts:135]: You have order and severityOrder here as an array. But all you care about is index in that array. 
Why don't convert those two into maps? Keys - severity levels, values - their assigned number/weights.
So then you will not need to use `indexOf` to search exact index, you just need to get element from the map.
Check out other columnsSortFunctionMap functions and you'll see how to do that, AssetGroupsComponent for example.

**Review by mkinchin-paladin — CHANGES_REQUESTED** [review]
PR needs some change

File: webapp/src/config/configurations.ts [hunk 3]
```diff
@@ -54,13 +54,13 @@
...
      },
      pacmanIssue: {
        CREATE_JIRA_TICKET_FOR_PACMAN_ISSUE: false, // Expected values: true || false || ''
        emailPacManIssue: {
          ISSUE_MAIL_TEMPLATE_URL: '',
-         ISSUE_EMAIL_FROM_ID: 'autofix@paladincloud.io',
+         ISSUE_EMAIL_FROM_ID: 'notify@paladincloud.io',
```
Comment (plyubich) [inline webapp/src/config/configurations.ts:63]: Why do you have the email address hardcoded on the UI? API should take care of figuring it out or hard coding in API side. These type of updates must be API driven!

If this requires an API change, we should engage the api team for support.

As stated before, keep such business logic out of UI and if you encounter future scenarios of such behavior please add a task to get that dependency removed from the UI.

**Review by plyubich — CHANGES_REQUESTED** [review]

**Comment by santhosh-challa** [comment]
@sidharthjain-zemoso Can we add a ticket for moving that email to API and move this?

**Comment by sidharthjain-zemoso** [comment]
> @sidharthjain-zemoso Can we add a ticket for moving that email to API and move this?

Yes. We have a backend ticket for this.

**Comment by santhosh-challa** [comment]
@plyubich as we added a ticket for API change, I am merging this.

**Pull request merged**